frontend/src/summary_charts.pug

```diff
@@ -88,8 +88,7 @@
         ) {{ getPercentile(i) }} %
 
       .summary-chart__ramp(
-        onclick="viewClick(event)",
```

Comment:
Why removed this? Am unable to `cmd-cliick` to zoom into ramps anymore.

Reply:
the click is controlled by opentabzoom. and the feature is working on my computer.
Can help me check if it is due to the mac system? anything printed on your console?

Reply:
Could be a possibility. Doens't work for me too.

Reply:
Fixed. I forgot to change this.isMacintosh to window.isMacintosh